models/Topic.js

```diff
@@ -0,0 +1,25 @@
+const db = require('../db/connection.js')
+
+class Topic {
+  static async findMany() {
+    const result = await db.query(`
+      SELECT * FROM topics
+    `)
+
+    return result.rows
+  }
+
+  static async findById(id) {
+    const result = await db.query(
+      `
+      SELECT * FROM topics
+      WHERE id = $1
+    `,
+      [id]
+    )
+
+    return result.rows[0]
+  }
+}
+
```

Comment:
Putting your models in a class like this is a bit overkill because it's recreating exactly what happens when we put things on the module.exports property. If you want to continue using this pattern though just keep it consistent throughout the week